client/src/components/Schedule/TableView/TableView.tsx

```diff
@@ -186,27 +144,12 @@ export function TableView({
         await courseService.deleteCourseEvent(id);
       }
 
-      await refreshData();
+      refreshData();
```

Comment:
wondering why remove `await`

Reply:
Because  refreshData returns void

Reply:
but it is not about return value, it is about waiting when all operations in function are completed, no?

Reply:
it returns void, not Promise<void>. Why should be wait ?

Reply:
https://github.com/streamich/react-use/blob/master/src/useAsyncRetry.ts

refreshData it's retry function in useAsyncRetry.
So when we call it we just trigger component re-render because of this hook state has changed.

Reply:
understood. now I see it was changed to retry. thx